avro/src/main/java/com/fasterxml/jackson/dataformat/avro/jsr310/AvroJavaTimeModule.java

```diff
@@ -0,0 +1,61 @@
+package com.fasterxml.jackson.dataformat.avro.jsr310;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.core.json.PackageVersion;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.dataformat.avro.jsr310.deser.AvroInstantDeserializer;
+import com.fasterxml.jackson.dataformat.avro.jsr310.deser.AvroLocalDateDeserializer;
+import com.fasterxml.jackson.dataformat.avro.jsr310.deser.AvroLocalDateTimeDeserializer;
+import com.fasterxml.jackson.dataformat.avro.jsr310.deser.AvroLocalTimeDeserializer;
+import com.fasterxml.jackson.dataformat.avro.jsr310.ser.AvroInstantSerializer;
+import com.fasterxml.jackson.dataformat.avro.jsr310.ser.AvroLocalDateSerializer;
+import com.fasterxml.jackson.dataformat.avro.jsr310.ser.AvroLocalDateTimeSerializer;
+import com.fasterxml.jackson.dataformat.avro.jsr310.ser.AvroLocalTimeSerializer;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.time.ZonedDateTime;
+
+/**
+ * A module that installs a collection of serializers and deserializers for java.time classes.
+ */
+public class AvroJavaTimeModule extends SimpleModule {
+
+    private static final long serialVersionUID = 1L;
+
+    public AvroJavaTimeModule() {
+        super(PackageVersion.VERSION);
```

Comment:
Since we do not want to use "parent" module name related to Avro package, will want to pass both module name and version: latter is same as parent version. So:

   super(getClass().getName(), PackageVersion.VERSION);

I think.

Reply:
Changed to: 
`super(AvroJavaTimeModule.class.getName(), PackageVersion.VERSION);`


I took inspiration from `IonJavaTimeModule` -> copy/pasting it.
BTW: `IonJavaTimeModule` is still using incorrect `PackageVersion`, see my comment https://github.com/FasterXML/jackson-dataformats-binary/commit/e78d78ab05b2d183b7bdf2da46ea6d3545d97475#r52273882

Reply:
Ah! Thank you, will fix that one.